src/extension/src/ActionHandler.py

```diff
@@ -219,22 +208,14 @@ def enable(self):
             self.logger.log_error("Error occurred during extension enable. [Error={0}]".format(repr(error)))
             return Constants.ExitCode.HandlerFailed
         finally:
-            if stdout_file_mirror is not None:
-                stdout_file_mirror.stop()
-            if file_logger is not None:
-                file_logger.close()
+            self.close_file_logger(file_logger, stdout_file_mirror)
 
     def disable(self):
         stdout_file_mirror = None
         file_logger = None
 
         try:
-            log_file_name = datetime.datetime.utcnow().strftime("%Y-%m-%d_%H-%M-%S") + "_Disable"
-            file_logger = self.utility.create_log_file(self.ext_env_handler.log_folder, log_file_name)
-            if file_logger is not None:
-                self.logger.file_logger = file_logger
-                stdout_file_mirror = StdOutFileMirror(file_logger)
-
+            file_logger, stdout_file_mirror = self.setup_file_logger(action="Disable")
```

Comment:
Strings serving functional purposes in code should ideally be mapped to constants.

Reply:
Applies to every action - not leaving comments for each.